test/simulator/timestamp/src/include/connection_simulator.h

```diff
@@ -40,12 +44,27 @@ class connection_simulator {
     static connection_simulator &get_connection();
     session_simulator *open_session();
     int query_timestamp() const;
-    int set_timestamp() const;
+    int set_timestamp(std::string config);
     ~connection_simulator();
+    uint64_t
+    get_oldest_ts()
+    {
+        return oldest_ts;
```

Comment:
Best to define this in connection_simulator.cpp

Reply:
My preference is to have it as a one liner getter here but since s_all doesn't allow that then I can move it to the source file.